bluetooth_mesh/messages/silvair/network_diagnostic_server.py

```diff
@@ -112,20 +110,20 @@ class NetworkDiagnosticSetupServerSubOpcode(IntEnum):
 NetworkDiagnosticServerSubscriptionStatus = Struct(
     "destination" / UnicastUnassignedGroupAddress,
     "period" / Int16ul,
-    "max_record_count" / Const(MAX_RECORD_COUNT, Int8ul),
+    "max_record_count" / Int8ul,
```

Comment:
?

Reply:
This will cause crash when number of supported records in the firmware will change. We cannot expect that max record count will be always 32.